### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by misscoded**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [x] Documentation

### Summary

This PR updates the README to better reflect the current development experience.

**Note: the URLs referenced will need to be updated, both in the README and the code comments. Another task for another day.**

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,25 @@ $ slack create my-app -t slack-samples/deno-starter-template
...
+$ slack run
+
+Connected, awaiting events
+```
+
+To stop running locally, press `<CTRL> + C` to end the process.
+
+## Creating Triggers
+
+When you run or deploy your project, the CLI will prompt you to create a new
```
Comment (filmaj) [inline README.md:64]: Maybe to make it slightly clearer / push to action, we monospace the two CLI action words?

```suggestion
When you `run` or `deploy` your project, the CLI will prompt you to create a new
```

File: README.md [hunk 2]
```diff
@@ -44,7 +44,25 @@ $ slack create my-app -t slack-samples/deno-starter-template
...
+trigger if one is found in the `triggers/` directory.
```
Comment (filmaj) [inline README.md:65]: I think this behaviour also only happens if no trigger exists for the particular app instance (app+workspace).

Worth calling that out here or no?

File: README.md [hunk 3]
```diff
@@ -128,37 +128,40 @@ app to Slack hosting using `slack deploy`:
 $ slack deploy
 ```
 
-After deploying, [create a new link trigger](#create-a-link-trigger) for the
-production version of your app (not appended with `(dev)`). Once the trigger is
-invoked, the workflow should run just as it did in when developing locally.
+After deploying, you'll be prompted to
```
Comment (filmaj) [inline README.md:131]:
```suggestion
After deploying for the first time, you'll be prompted to
```

File: README.md [hunk 4]
```diff
@@ -128,37 +128,40 @@ app to Slack hosting using `slack deploy`:
...
+[create a new link trigger](#creating-triggers) for the production version of
+your app. When that trigger is invoked, the workflow should run just as it did
+in when developing locally (but without requiring your server to be running).
```
Comment (filmaj) [inline README.md:134]: Just me or is the wording in this sentence a bit weird? "Just as it did in when developing locally..."

**Review by filmaj — APPROVED** [review]
Huge improvement! Thanks for doing this. Also a great template for the other samples' README updates.

Left a few comments, thanks for doing this.

File: README.md (hunk 4 above)
Comment (vegeris) [reply README.md:134]: The 'in' may be superfluous; `Just as it did when developing locally...`

File: README.md [hunk 5]
```diff
@@ -62,44 +80,26 @@ that Shortcut URLs will be different across each workspace, as well as between
 
 When creating a trigger, you must select the workspace and environment that
 you'd like to create the trigger in. Each workspace has a local development
```
Comment (vegeris) [inline README.md:82]:
```suggestion
you'd like to create the trigger in. Each workspace can have a local development
```

File: README.md (hunk 2 above)
Comment (misscoded) [reply README.md:65]: Oh, really? So, if someone evolves the app and later adds a trigger, it won't get picked up?

File: README.md (hunk 4 above)
Comment (misscoded) [reply README.md:134]: Totally right, great catch, will edit!

File: README.md (hunk 2 above)
Comment (misscoded) [reply README.md]: Did a bit of editing and rearranging. Let me know what you think, but the nod added is:

> When you `run` or `deploy` your project for the first time, the CLI will prompt
you to create a trigger if one is found in the `triggers/` directory. For any
subsequent triggers added to the application, each must be
[manually added using the `trigger create` command](#manual-trigger-creation).

Comment (filmaj) [reply README.md]: Perfect, thank you!

**Review by hello-ashleyintech — COMMENTED** [review]
Looking great! 🎉 Left a few small thoughts / comments

**Pull request merged**